server/src/controllers/auth.ts

```diff
@@ -0,0 +1,80 @@
+// SPDX-FileCopyrightText: 2022 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import User from '../database/models/user';
+import argon from 'argon2';
+
+export type PlainPassword = string;
+export enum UserRole {
+  Student = 'STUDENT',
+  Teacher = 'TEACHER',
+  Admin = 'ADMIN',
```

Comment:
The database enum in `server/src/database/models/courseRole.ts` calls this `'SYSADMIN'`, one of the two should be changed to be consistent.